modules/P05_firmware_bin_extractor.sh

```diff
@@ -54,13 +57,14 @@ P05_firmware_bin_extractor() {
   deb_extractor
   ipk_extractor
 
-  BINS=$(find "$FIRMWARE_PATH_CP" "${EXCL_FIND[@]}" -type f -executable | wc -l )
-  UNIQUE_BINS=$(find "$FIRMWARE_PATH_CP" "${EXCL_FIND[@]}" -type f -executable -exec md5sum {} \; | sort -u -k1,1 | wc -l )
+  BINS=$(find "$FIRMWARE_PATH_CP" "${EXCL_FIND[@]}" -xdev -type f -executable | wc -l )
+  UNIQUE_BINS=$(find "$FIRMWARE_PATH_CP" "${EXCL_FIND[@]}" -xdev -type f -executable -exec md5sum {} \; 2>/dev/null | sort -u -k1,1 | cut -d\  -f3 | wc -l )
   if [[ "$BINS" -gt 0 || "$UNIQUE_BINS" -gt 0 ]]; then
     print_output ""
     print_output "[*] Found $ORANGE$UNIQUE_BINS$NC unique executables and $ORANGE$BINS$NC executables at all."
   fi
 
+  echo -e "\\n[*] HTML_REPORT:$HTML_REPORT" >> "$LOG_FILE"
```

Comment:
Additional to [the previous comment](https://github.com/e-m-b-a/emba/pull/87/files#r603889748):

`echo -e "\\n[*] HTML_REPORT:$HTML_REPORT" >> "$( get_log_file )"`